manifests/c/CrestronElectronicsInc/CrestronToolbox/3.1130.0019.0/CrestronElectronicsInc.CrestronToolbox.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: CrestronElectronicsInc.CrestronToolbox
+PackageVersion: 3.1130.0019.0

```

Comment:
InstallerType: "exe"
InstallModes:
- "/silent"

Reply:
Hi @botbeans,

Was this comment supposed to be wrapped in "suggestion" marks? It seems like a good addition to this manifest.